<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasketDialog</class>
 <widget class="QDialog" name="BasketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basket</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Basket {
background-color: rgb(164, 164, 164);
border:0px solid white ;
border-radius:4px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="Basket" native="true">
     <property name="styleSheet">
      <string notr="true">#packageList, #extraList {
	border:1px solid #888;
	border-radius:3px;
	background-color:rgba(252,252,252,100);
    color:rgba(20,20,20);
}
QLabel { color:rgb(20,20,20); }</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>26</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget{
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);

border:0px solid #666;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom:1px solid #999;
}
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="infoLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Selected package(s) for install:</string>
           </property>
           <property name="indent">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="styleSheet">
            <string notr="true">color:rgb(20,20,20);</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,2" rowminimumheight="0,0,0,0,0,0,0,0" columnminimumwidth="0,1,2">
         <item row="0" column="0" colspan="3">
          <widget class="PackageView" name="packageList">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>52</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>52</number>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QLabel" name="extrasLabel">
           <property name="text">
            <string>Extra dependencies of the selected package(s) that are also going to be installed:</string>
           </property>
           <property name="indent">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="PackageView" name="extraList">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>52</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>52</number>
           </attribute>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Total Size:</string>
           </property>
           <property name="indent">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="totalSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;b&gt;2.2 MB&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2" rowspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Clear Basket</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="actionButton">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Install Package(s)</string>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="downloadSizeLabel">
           <property name="text">
            <string>Download Size:</string>
           </property>
           <property name="indent">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="downloadSize">
           <property name="text">
            <string>&lt;b&gt;2.0 MB&lt;/</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>3</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PackageView</class>
   <extends>QTableView</extends>
   <header location="global">packageview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>packageList</tabstop>
  <tabstop>extraList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
